acados/ocp_nlp/ocp_nlp_sqp.c

```diff
@@ -607,7 +685,8 @@ static bool ocp_nlp_soc_line_search(ocp_nlp_config *config, ocp_nlp_dims *dims,
             ocp_qp_res_compute_nrm_inf(work->qp_res, mem->stat+(mem->stat_n*(sqp_iter+1)+7));
     }
 
-    if (nlp_opts->print_level > sqp_iter + 1)
+    // if (nlp_opts->print_level > sqp_iter + 1)
+    if (nlp_opts->print_level > 5)
```

Comment:
I undid the changes, since they were unintended. But still it seems weird, why you would want a condition that is dependent on the current iteration. Maybe it makes more sense to replace this condition by a condition `print_qp_in` that can be adjusted for a respective iteration